web-new/src/app/globals.css

```diff
@@ -5,10 +5,36 @@
 @layer base {
   body {
     @apply bg-brand-80 text-brand-20;
-    @apply font-secondary;
+    @apply font-secondary text-body-1;
     box-sizing: border-box;
     background-image: url("../../public/images/bg-body.png");
     background-size: cover;
     background-repeat: no-repeat;
+    background-attachment: fixed;
+  }
+}
+
+
+@layer components {
```

Comment:
We should not apply styles this way, it goes against tailwind best practise

Reply:
to be added as separate PR by urself later 🙏